cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -9,8 +9,10 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
 
 {-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# OPTIONS_GHC -fwarn-incomplete-patterns #-}
```

Comment:
Why is this needed?

Reply:
It shouldn't be. I was trying to understand why GHC did not complain about: https://github.com/IntersectMBO/cardano-cli/pull/728/files#diff-a38343c1825e6e334224d0a8a1ba78a458652970f00c54f5fb0d040be28f822eR787

Reply:
:smiling_face_with_tear: 
```
              ┃   │ ━━━━ stderr ━━━━
                ┃   │ 
                ┃   │ cardano-cli: user error (Pattern match failure in 'do' block at src/Cardano/CLI/EraBased/Run/Transaction.hs:787:3-26)
```

Reply:
I updated it.